lib/LEDStatus/LEDStatus.h

```diff
@@ -0,0 +1,44 @@
+#include <Arduino.h>
+
+#ifndef _LED_STATUS_H
+#define _LED_STATUS_H
+
+class LEDStatus {
+    public:
+        enum LEDMode {

```

Comment:
Should we make this an `enum class`?  My guess is that your C++ is stronger than mine, so please feel free to educate me if there's not a good reason for this.

Reply:
Yes, that is better.  I'm an old school C++ guy (haven't used C++ in countless years) so getting back to speed with this project!  I'll make that change.